cirq/contrib/noise_models/noise_models.py

```diff
@@ -142,3 +145,121 @@ def noisy_moment(self, moment: 'cirq.Moment',
                 moment,
                 ops.Moment(self.qubit_noise_gate(q) for q in system_qubits)
             ]
+
+
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows probabilities to be
+    specified separately for each qubit.
+
+    This simulates asymmetric readout error. The noise is structured
+    so the T1 decay is applied, then the readout bitflip, then measurement.
+    Note that T1 decay is only applied to measurement, not other gates.
+
```

Comment:
How does the noise model act on moments without measurement?

Reply:
Non-measurement gates will have depolarizing noise applied; idle qubits will remain unaffected. Updated docstring.